R/data.R

```diff
@@ -114,13 +114,11 @@
 #' @details
 #' Use this example to understand:
 #' - How to work with negative corrections in delay distributions
-#' - The difference between `preprocess = NULL` (preserves negatives) and
-#'   `preprocess = preprocess_negative_values` (redistributes)
+#' - The impact of preprocessing negative values on delay estimation
 #' - How PMFs and CDFs behave with systematic downward corrections
 #'
 #' @seealso
 #' - [example_reporting_triangle] for a clean example without corrections
-#' - [estimate_delay()] with `preprocess` parameter
```

Comment:
Should we add `[baselinenowcast.reporting_triangle()]` with `preprocess` parameter as this describes the `preprocess` parameter?

Reply:
Have added